src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -63,11 +64,11 @@ public class GridPanelTest extends GridPanelBaseTest
     // Column names.
     private static final String FILTER_NAME_COL = "Name";
     private static final String FILTER_EXPDATE_COL = "Expiration Date";
-    private static final String FILTER_STRING_COL = TestDataGenerator.randomFieldName("Str", 0, 5);
-    private static final String FILTER_INT_COL = TestDataGenerator.randomFieldName("Int", 0, 5);
-    private static final String FILTER_EXTEND_CHAR_COL = TestDataGenerator.randomFieldName("\u0106\u00D8\u0139", 0, 5);
-    private static final String FILTER_BOOL_COL = TestDataGenerator.randomFieldName("Bool", 0, 5);
-    private static final String FILTER_DATE_COL = TestDataGenerator.randomFieldName("Date", 0, 5);
```

Comment:
`TestDataGenerator.randomFieldName` has an exclusion parameter, could that be an option here?

Reply:
Possibly. I was getting all sorts of failures across numerous tests and didn't think I'd be able to squash them all before the weekend.